index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"> 
+    <title>Personal Portfolio Website</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+
+        </header>
+        <header class="menu">
+            <div class="menu-logo">
+                <a>Welcome</a>
+            </div>
+            <nav class="menu-button">
+                <ul>
+                    <li class="bar"> </li>
+                    <li class="bar"> </li>
+                    <li class="bar"> </li>
+                </ul>
+            </nav>
+        </header>
+        <nav class="nav-menu">
+            <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+        <section class="hero">
+            <section class="headline">
+                <div class="place primary">
+                    <h2> Hey there. I’m Alvin <br><span class="green"> I’m a software developer </span></h2>
+                </div>
```

Comment:
Reduce the white space on the top and add some pace between the name and the title
![right](https://user-images.githubusercontent.com/58376020/125834855-e337ceda-f84c-4fb0-89a9-b22b10f9945d.png)

Reply:
This issue was well resolved ✔️